sklearn/utils/_plot.py

```diff
@@ -0,0 +1,53 @@
+from . import check_consistent_length, check_matplotlib_support
+from .multiclass import type_of_target
+from .validation import _check_pos_label_consistency
+from ._response import _get_response_values_binary
+
+
+class _BinaryClassifierCurveDisplayMixin:
+    """Mixin class to be used in Displays requiring a binary classifier.
+
+    The aim of this class is to make some validations regarding the estimator and the
+    target and gather the response of the estimator.
+    """
+
+    def plot(self, *, name=None):
+        check_matplotlib_support(f"{self.__class__.__name__}.plot")
+        name = self.estimator_name if name is None else name
+        return name
```

Comment:
I find it confusing to name this function ``plot``, and if displays to do ``name = super().plot``. I don't have a good name idea as of now but let's try to come up with a better one